### ISSUE TITLE
Redirect Request: VAMC Leadership pages

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-team#26475]
### Instructions
- Requests for URL changes/redirects need to be submitted **AT LEAST** 2 weeks in advance.  Some changes/redirects take significant amount of time and coordination to implement, so start the process as soon as you know you will need one. 
- This issue will be used from initial request through implementation of the redirect to ensure all individuals working on this are notified of status updates.  Please do not create multiple issues to track differnt steps.
- It is your responsibility to notify VA stakeholders of pending redirect.  

### Description
*This is a detailed description of the request. It should include background information and reason for the request.*

### Requesting team info
*Indicate what team and who the primary contact on that team is*

### Implementation date
Date new URL(s) will be live: *enter date new pages/URLs will be live*

TBD, sometime in early July?

Please indicate when the change/redirect(s) needs to be implemented"
- [ ] On the same date the new URLs launch
- [ ] Can happen within 1-5 business days after new URLs launch
- [x] Other - Please indicate timing : Within an hour or two of changes in CMS to Leadership page URLs

**The requesting team is responsible for notifying the group working on this issue if the target date changes. They are also responsible for ensuring the destination URLs are implemented correctly and live at the time the redirects are deployed.**

### Redirects needed

Current URL  |  Redirect Destination or New URL
---  |  ---
pittsburgh-health-care/about/leadership | pittsburgh-health-care/about-us/leadership
wilkes-barre-health-care/about/leadership | pittsburgh-health-care/about-us/leadership
altoona-health-care/about/leadership | pittsburgh-health-care/about-us/leadership
wilmington-health-care/about/leadership | pittsburgh-health-care/about-us/leadership

[possibly others as new VAMCS go to full prod ... could this be wildcarded?]

### Definition of done
- [ ] Requesting team: Above information is provided and issue is tagged and assigned appropriately 
- [ ] Requesting team: All appropriate VA stakeholders are notified of pending redirect 
- [ ] VSP IA: Request is vetted and documented and implementation plan is clear - *@ VSP IA*
- [ ] VSP IA: Request is assigned to appropriate team for implementation - *@ VSP IA team*
- [ ] Implementation team: Implementation team completes work 
- [ ] Implementation team: Implementation pushed live and redirect is validated in production
- [ ] Requesting team: Ticket is closed

**Comment by mnorthuis** [comment]
@kevwalsh I don't think we have to worry about any other VAMCs because there are no others live yet.  We will only need to do formal redirects if a health system is fully live (out of dual state and legacy site is redirected).   So I'm guessing this only impacts Pitts...how did we miss these in our other list???

**Comment by mnorthuis** [comment]
@kevwalsh Looking at analytics for the month of June for these leadership pages
- page views are incredibly low
- Under 20% of the page views are entrances for users - in other words, that's the page they entered the site on.  This gives us an idea of external links and SEO value.   For this, most users are getting to these pages from another page on the site, primarily the "About us" page.  
- Besides Pitts, the other sites are not fully live yet and are getting little to no traffic. 

Given the above info, I don't see the need to implement a redirect for these pages.  My opinion is that the URL for these pages can change without a redirect and will have very little impact to our users.

**Comment by kevwalsh** [comment]
@mnorthuis Just to be clear, no redirect for Pittsburgh either?

**Comment by mnorthuis** [comment]
@kevwalsh I don't feel its necessary.

**Comment by mnorthuis** [comment]
@kevwalsh Was there agreement that a redirect was not needed?  I'm assuming we can close this issue?

**Comment by kevwalsh** [comment]
there was! thanks Mikki.
